tools/samba/illumina_suite/swarm_clustering.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="swarm_clustering" name="Swarm clustering" version="@TOOL_VERSION_SWARM@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
+    <description>Clustering of AVSs using swarm</description>
+    <xrefs>
+        <xref type="bio.tools"></xref>
+    </xrefs>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="figaro_edam"/>
+    <expand macro="swarm_requirements"/>
+    <required_files>
+        <include type="literal" path="macro.xml"/>
+    </required_files>
+    <command detect_errors="exit_code"><![CDATA[
+        CPUS='$cpu'
+        ASV_SEQS_FASTA='$asv_seq'
+        SWARM_ASV_SEQS_FA='$swarm_asv_seq'
+        TMP_SWARM_CLUSTER_LIST='./tmp_dir'
+        FINAL_SWARM_ASV_SEQS_FA='./output_dir'
+        LOGCMD='./log_dir'
+
+        # Run swarm
+        CMD="swarm -t ${CPUS} -f -l swarm.log -w ${SWARM_ASV_SEQS_FA} ${ASV_SEQS_FASTA} > ${TMP_SWARM_CLUSTER_LIST} "
+        echo ${CMD} > ${LOGCMD}
+        eval ${CMD}
+
+        # Reformat swarm sequence header FASTA file
+        CMD="cut -d '_' -f1 ${SWARM_ASV_SEQS_FA} > ${FINAL_SWARM_ASV_SEQS_FA}"
+        echo ${CMD} >> ${LOGCMD}
+        eval ${CMD}
+
+    ]]></command>
+    <inputs>
+        <param name="cpu" type="integer" value="0"  label="CPU" help="TODO" />
```

Comment:
This should be put as default (did not find exactly how to do it in frogs fro now)